### DISCUSSION TITLE
Per World Chat and Tablist

### DISCUSSION THREAD

**Discussion by ghost** [Zrips/CMI#4197] [Feature Requests]
This would be a very useful feature especially because there are hardly any alternatives to plugins that can do this. If CMI could do that it would be very nice.  Optimal for multiverse setups.

**Comment by JetSt0rm** [comment]
I second this. I've consolidated many plugins into CMI but have to keep chatmanager for it's multiworld functionality. Everything else it does is handled by CMI.
